packages/react-ai/src/components/AIConversation/types.ts

```diff
@@ -88,7 +88,7 @@ type ResponseComponentPropMap = {
 };
 
 export type ResponseComponent = {
-  component: React.ComponentType;
+  component: React.ComponentType<any>;
```

Comment:
Note: we might want to make this a generic

Reply:
Will make a follow up to clean this up!